astro/src/content/docs/get-started/core-concepts/users.mdx

```diff
@@ -99,14 +99,28 @@ These are the default fields that are available when creating a user. You can ad
       Then only the setup password message will be sent to the user. Setting up the password using the message sent during this user create operation will verify the user’s email address or phone number if it is not already verified.
     </Aside>
     <Aside>
-      This is only available if the [Setup password option](/docs/get-started/core-concepts/tenants#template-settings) in the tenant Template settings is set up.
+      <InlineField>Send password setup to</InlineField> is only available if the following tenant settings are:
+      * [Setup password option](/docs/get-started/core-concepts/tenants#template-settings) - template settings must be set up
+      * [Password enabled](/docs/get-started/core-concepts/tenants#password) - set to `true`
```

Comment:
Did not say "only applies to version 1.99.9" here because thought that might be too busy. Might be best to explain that on the tenant docs, when https://linear.app/fusionauth/issue/ENG-2547/document-tenant-password-enabled-option is done